### PR TITLE
Change Diona dishes flavour

### PR DESCRIPTION
**Pull request by Ultrazundy**
* Changed diona roast taste from "salad" to "Diona delicacy" as part of a series of projects to create more diona dishes and make diona dishes tasty to tie in with lore. (I did it Caelphon).

### CODE REVIEW

**Comment by Ultrazundy** [comment]
!wip

**Comment by Caelphon** [comment]
Zundy the Turtle did it! Seems good tbh.

**Comment by Ultrazundy** [comment]
!wip
!review

**Comment by Ultrazundy** [comment]
!wip

**Comment by Ultrazundy** [comment]
!review

**Comment by alsoandanswer** [comment]
Roasted delicacy doesn't really mean anything, though.

**Comment by Ultrazundy** [comment]
> 
> 
> Roasted delicacy doesn't really mean anything, though.

I was gonna debate how we don't just set every salad's taste to "salad" and go into how salad is super generic but we actually _do_ have every salad having that taste pretty much. That and some dishes like bear stew missing their key ingredient in their taste desc. How about "dionae" for the taste?

**Comment by Chada-1** [comment]
Why not 'Diona Delicacy' or something like that? For the Diona taste, then have the roasted just be in the dish name etc or something? Otherwise this is v. cook

**Comment by Ultrazundy** [comment]
> 
> 
> Why not 'Diona Delicacy' or something like that? For the Diona taste, then have the roasted just be in the dish name etc or something? Otherwise this is v. cool
> 
> Edit: Mispelled cool as cook 🐒

Perfect, changing now.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -3987,7 +3987,7 @@
 	trash = /obj/item/trash/plate
 	filling_color = "#75754B"
 	center_of_mass = list("x"=16, "y"=7)
-	nutriment_desc = list("salad" = 3)
+	nutriment_desc = list("Dionae delicacy" = 3)
```
Comment (alsoandanswer) [inline code/modules/reagents/reagent_containers/food/snacks.dm:3990]:
```suggestion
	nutriment_desc = list("dionae delicacy" = 3)
```
keep it lower case like everything else thanks

**Pull request merged**